typescript/sdk/src/hook/HyperlaneHookDeployer.ts

```diff
@@ -182,11 +182,11 @@ export class HyperlaneHookDeployer extends HyperlaneDeployer<
       origin: chain,
       nativeBridge: l2Messenger,
     };
-    const opstackIsm = (await this.ismFactory.deploy(
-      config.destinationChain,
-      ismConfig,
-      chain,
-    )) as OPStackIsm;
+    const opstackIsm = (await this.ismFactory.deploy({
+      chain: config.destinationChain,
+      config: ismConfig,
+      origin: chain,
```

Comment:
The naming here could be better I think. Should `chain` be `destination` so it's not confused with `origin`?